client/src/components/Overview/index.jsx

```diff
@@ -15,19 +17,31 @@ class Overview extends React.Component {
 
   render() {
     const { allStyles } = this.state;
+
     let images;
     let stockKeepingUnit;
+    let originalPrice;
+    let salePrice;
+
     allStyles.forEach((style) => {
       if (style['default?'] === true) {
         images = style.photos;
         stockKeepingUnit = style.skus;
+        originalPrice = style.original_price;
+        salePrice = style.sale_price;
       }
     });
 
     return (
       <div className={styles.overview}>
         <div className={styles.imageGallery}><ImageGallery images={images} /></div>
```

Comment:
one thing I notice everyone doing is putting Components inside of divs, then having to style the divs and the components.  Why not make your life easier?  The better practice would be to completely replace the div with your component and style one thing (where possible of course).

```suggestion
        <ImageGallery images={images} />
```

Reply:
ahhh I see! good idea, it will decrease the number of divs that I need to style for one sub-component, I will definitely incorporate this :)

Reply:
Yes, Phil made this suggestion to me and I think it has made my life easier.